src/mame/pinball/criter75.cpp

```diff
@@ -0,0 +1,133 @@
+  |__    |_F9334PC_|  |_SN74155N|  1|     |11  |_________|  |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |__|   |_F9334PC_|  |_SN74151N|  2|     |10  |_PCF7418_|  |_________|     |
+  |__     __________   __________   |     |     __________   __________     |
+  |--|   |_SN7404N_|  |_SN7400N_|  3|     |9     o o o o    |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |_SN74193N|  |_SN74155N|  4|     |8   |_MM74C02N|  |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |_SN7404N_|  |_SN7400N_|  5|     |7   |_SN7404N_|  |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |__|   |_SN74153N|  |_SN7404N_|  6|     |6   |_SN7400N_|  |_SN74193N|     |
+  |__     __________   __________   |     |     __________   __________     |
+  |--|   |_SN7404N_|  |_SN7400N_|  7|     |5   |_SN7402N_|  |_SN7425N_|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |_F9334PC_|  |_SN74163N|  8|     |4   |_SN7493AN|  |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |_SN74151N|  |_PROM_A9_|  9|     |3   |_SN7404N_|  |_SN7474N_|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |__9334PC_|  |_PROM_A10| 10|     |2   |__DIPSx8_|  |_________|     |
+  |--|    __________   __________   |     |     __________   __________     |
+  |--|   |_N74107A_|  |_SN74163N| 11|     |1   |_________|  |_________|     |
+  |--|    __________   __________   |     |_________________________________|
+  |--|   |_SN7493AN|  |_SN7493AN| 12|
+  |--|    __________   __________   |
+  |__|   |_SN7404N_|  |_SN7493AN| 13|
+  |_________________________________|
+
+*/
+
+#include "emu.h"
+
+#include "machine/netlist.h"
+
+#include "netlist/devices/net_lib.h"
+
+namespace {
+
+class criter75_state : public driver_device
+{
+public:
+	criter75_state(const machine_config &mconfig, device_type type, const char *tag)
+	: driver_device(mconfig, type, tag),
+		m_maincpu(*this, "maincpu")
+	{
+	}
+
+	void criter75(machine_config &config);
+
+private:
+	// devices
+	required_device<netlist_mame_device> m_maincpu;
+
+	// driver_device overrides
+	virtual void machine_start() override ATTR_COLD;
+	virtual void machine_reset() override ATTR_COLD;
+};
+
+static NETLIST_START(criter75)
+{
+	SOLVER(Solver, 48000)
+	PARAM(Solver.ACCURACY, 1e-4)
+
+	// schematics
+	//...
+}
+
+void criter75_state::machine_start()
+{
+}
+
+void criter75_state::machine_reset()
+{
+}
+
+void criter75_state::criter75(machine_config &config)
+{
+	// basic machine hardware
+	NETLIST_CPU(config, m_maincpu, netlist::config::DEFAULT_CLOCK()).set_source(netlist_criter75);
+}
+
+ROM_START( criter75 )
+	ROM_REGION( 0x10000, "maincpu", ROMREGION_ERASE00 )
+	// There's no CPU ROMs
```

Comment:
Why do you have a CPU region for a system that has no CPU?

Reply:
Fixed, thanks!!!